index.html

```diff
@@ -44,73 +43,82 @@ <h1 class="item1 h1">I’m Kirubel. <br>Glad to see you!</h1>
     </ul>
   </section>
   <section class="work1">
-    <img class="work1-img" src="./images/Snapshoot Portfolio (1).png" alt="">
-    <h1>Tonic</h1>
+    <img class="work1-img" src="./images/port.png" alt="snapshot of website">
+    <p class="oldo">Tonic</p>
     <ul>
       <div class="work1-li1">
         <li class="work1-item1 work1-item3">CANOPAY</li>
         <li class="work1-item1 work1-item2">Back End Dev</li>
         <li class="work1-item1 work1-item2">2015</li>
       </div>
+      </ul>
        <p class="pp">A daily selection of privately personalized reads; no accounts or sign-ups required.</p>
+       <ul>
        <div class="work1-link">
-      <li>HTML</li>
-      <li>CSS</li>
-      <li>JavaScript</li>
+      <li >HTML</li>
+      <li class="newlin">CSS</li>
+      <li class="newlin">JavaScript</li>
       </div>
       <button>See Project</button>
     </ul>
-    <img src="./images/Snapshoot Portfolio.png" alt="">
-    <h1>Multi-Post<br>Stories</h1>
+    <img src="./images/Snapshoot.png" alt="snapshot of website">
+    <p class="oldo">Multi-Post<br>Stories</p>
     <ul>
       <div class="work1-li1">
         <li class="work1-item1 work1-item3">CANOPAY</li>
         <li class="work1-item1 work1-item2">Back End Dev</li>
         <li class="work1-item1 work1-item2">2015</li>
       </div>
+      </ul>
        <p class="pp">A daily selection of privately personalized reads; no accounts or sign-ups required.</p>
+       <ul>
        <div class="work1-link">
-      <li>HTML</li>
-      <li>CSS</li>
-      <li>JavaScript</li>
+      <li >HTML</li>
+      <li class="newlin">CSS</li>
+      <li class="newlin">JavaScript</li>
       </div>
+    </ul>
```

Comment:
- [ ] As suggested by the previous reviewer, I would like to recommend you to give a closer look into the figma design and make your design look almost exactly. This includes, color, font-size, font-weight, font-style, margin, padding background-color among others. I would love to encourage your to work your design alongsize the figma design element by element.

### Your Design
![your-works](https://user-images.githubusercontent.com/26597432/227711654-e680cd5d-ec81-414e-b893-1af3d487756c.png)

### figma Design
![temp-works](https://user-images.githubusercontent.com/26597432/227711663-cfbe3051-e449-4591-a0ef-ae2a138e6483.png)